docs/evaluate/how-it-works.mdx

```diff
@@ -0,0 +1,90 @@
+---
+id: how-it-works
+title: How Temporal Works
+sidebar_label: How It Works
+description: This guide provides a short overview of How Temporal Works.
+toc_max_heading_level: 4
+keywords:
+  - temporal
+  - evaluate-temporal
+  - how-temporal-works
+tags:
+  - temporal
+  - evaluate-temporal
+  - how-temporal-works
+---
+Temporal offers an entirely new way to build scalable and reliable applications.
+
+## Build Invincible Apps 
+
+Temporal ensures your applications are reliable by default, handling everything from minor network outages to significant issues like unexpected reboots or hardware failures.
+By offloading the responsibility of failure management from the application to the platform, this removes the need for extensive coding, testing, and maintenance tasks.
+
+This shift simplifies the development process and enables your application to seamlessly recover from disruptions.
+It does so by automatically reconstructing its previous state.
+Your application logic can be recovered, replayed, or paused from any previous point in the application process.
+
+![Describing a Temporal Application - How It Works](/img/how-it-works/hiw-yourapplicationclient.png)
+
+### Temporal Application
+
+A Temporal application is the code you write, comprised of Workflow and Activities. 
+A Temporal Workflow defines your overall business logic. That business logic might involve moving money between bank accounts, processing orders, deploying cloud infrastructure, training an AI model, or something else entirely.
+The Worker runs the Workflow and Activity code.
+
+![Workflow Function Execution - How It Works](/img/how-it-works/hiw-workflowexecution.png)
+
+### Activity
+Activities represent code (either a functions or method) that handle tasks prone to errors or failure, such as network requests, calling a service, sending an email, or generating random numbers.
+Activities are invoked in Workflow code and the Temporal Service coordinates with the application to execute them.
+
+![Continuing a Workflow - How It Works](/img/how-it-works/hiw-continuingtheworkflow.png)
+
+### Temporal Service
+
+Temporal consists of two components, the [Temporal Service](/evaluate/major-components#temporal-service) and an SDK in your preferred language such as Go, .NET, Java, Python, PHP, or Typescript.
+The Temporal Service logs the progress and state of your application.
```

Comment:
"logs the progress and state" is kind of off to me in a Temporal Context. How about: "The Temporal Service records every event in the Workflow’s lifecycle, so that the Workflow can continue from the last successful state in the case of a failure" or something like that

Reply:
I edited this after receiving this feedback so these points are mentioned now.